aws/resource_aws_instance.go

```diff
@@ -576,6 +576,22 @@ func resourceAwsInstance() *schema.Resource {
 					},
 				},
 			},
+
+			"enclave_options": {
+				Type:     schema.TypeList,
+				Optional: true,
```

Comment:
To be safe and prevent unexpected differences for operators should the EC2 API return this information with existing configurations not having it, let's add `Computed` here:

```suggestion
				Optional: true,
				Computed: true,
```

Reply:
I also switched the `enabled` field to `Computed: true`. This means that `aws_instance` resources that originally had enclaves enabled won't force a new resource if the `enclave_options` block is removed, which seems like a reasonable behavior.

Reply:
The top level `Computed: true` on configuration blocks prevents differences from showing up in nested attributes if the configuration is not provided even if it does not match the default. Since there is one nested attribute though, it is probably okay since most operators would likely configure both the block and the attribute.